src/routes/docs/tutorials/android/step-3/+page.markdoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial
+title: Setup Appwrite
+description: Learn to build an Android app with no backend code using an Appwrite backend.
+step: 3
+difficulty: beginner
+---
+
+# Create project {% #create-project %}
+
+Head to the [Appwrite Console](https://cloud.appwrite.io/console).
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+If this is your first time using Appwrite, create an account and create your first project.
+
+Give your project a new name, and set **Project ID** to `recipewrite`.
+
+Then, under **Add a platform**, add a **Android app**. **Package Name** can be found in app level gradle file. 
+
+{% only_dark %}
+![Add a platform](/images/docs/quick-starts/dark/add-platform.png)
+{% /only_dark %}
+{% only_light %}
+![Add a platform](/images/docs/quick-starts/add-platform.png)
+{% /only_light %}
+
+You can skip the optional steps.
+
+# Connect your app with appwrite {% #connect-your-app-with-appwrite %}
+
+Create **Appwrite.java**, this will handle our app's interaction with Appwrite server.
+
+```java
```

Comment:
Also missing imports

Reply:
I am just providing the entire classes and methods and not caring about package names, imports, etc. That is not an issue because of modern text editors, and I believe they would just add to the noise in our tutorial. LMK if you feel otherwise.